validator/client/runner_test.go

```diff
@@ -83,7 +83,7 @@ func TestRunOnNextSlotUpdatesAssignmentsHandlesError(t *testing.T) {
 	testutil.AssertLogsContain(t, hook, "Failed to update assignments")
 }
 
-func TestRunOnNextSlotDeterminesRole(t *testing.T) {
+func TestRoleAt_OnNextSlot(t *testing.T) {
```

Comment:
I don't think we need the `On`, it sounds nicer w/o it ya?

Same for the below

Reply:
Yeah I wasn't sure if NextSlot was too vague or not. Will do!